performance-tests/commoncrawl-project/gradle.properties

```diff
@@ -0,0 +1,122 @@
+# These settings are used by Data Hub when
+# communicating with MarkLogic.
+# The values in this file are meant as project-wide settings.
+# You can override these properties for a specific environment
+# by creating a gradle-{environment}.properties file.
+# For example, to create a properties file for your prod environment create a file
+# named gradle-prod.properties.
+#
+# ....
+mlDHFVersion=5.3.0
```

Comment:
Do we still need this? if so we might need to update the version.

Reply:
This should be removed